iceoryx_binding_c/source/cpp2c_enum_translation.cpp

```diff
@@ -76,6 +80,8 @@ iox_WaitSetResult WaitSetResult(const iox::popo::WaitSetError value)
         return WaitSetResult_WAIT_SET_FULL;
     case WaitSetError::EVENT_ALREADY_ATTACHED:
         return WaitSetResult_EVENT_ALREADY_ATTACHED;
+    case WaitSetError::PROVIDED_HAS_TRIGGERED_CALLBACK_IS_UNSET:
```

Comment:
Judging by the wording I do not know whether this can even happen (can we unset the callback during waiting?). Has probably not much to do with the chunk header though. Weird error name anyway.

Reply:
yes, I just noticed that some enum values were missing in the translation and updated them